content/docs/intro/concepts/stack.md

```diff
@@ -61,24 +67,48 @@ test                                      2 weeks ago              121
 To select a stack that is part of an organization, use the fully-qualified stack name, either `orgName/stackName` or `orgName/projectName/stackName`:
 
 ```bash
-$ pulumi stack select acmecorp/prod
+$ pulumi stack select mycompany/prod
 
 $ pulumi stack ls
 NAME                                      LAST UPDATE              RESOURCE COUNT
-acmecorp/prod*                            4 hours ago              97
-acmecorp/staging                          4 hours ago              97
+mycompany/prod*                            4 hours ago              97
+mycompany/staging                          4 hours ago              97
 dev                                       n/a                      n/a
 ```
 
 ## Deploy a project
 
-To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "config" >}}) for the active stack.
+To deploy your project to the currently selected stack, run `pulumi up`. The operation uses the latest [configuration values]({{< relref "/docs/intro/concepts/config" >}}) for the active stack.
+
+{{% notes "info"%}}
+Your stack can distinguish between execution for `preview` versus for `update` by using [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
+{{% /notes %}}
 
-> **Note:** Your stack can distinguish between execution for preview versus for update by using [pulumi.runtime.isDryRun()]({{< relref "/docs/reference/pkg/nodejs/pulumi/pulumi/runtime#isDryRun" >}}).
+## Stack tags
+
+Stacks have associated metadata in the form of tags, with each tag consisting of a name and value. A set of built-in tags are automatically assigned and updated each time a stack is updated (such as `pulumi:project`, `pulumi:runtime`, `pulumi:description`, `gitHub:owner`, `gitHub:repo`, `vcs:owner`, `vcs:repo`, and `vcs:kind`). To view a stack's tags, run [`pulumi stack tag ls`]({{< relref "/docs/reference/cli/pulumi_stack_tag_ls" >}}).
```

Comment:
Worth using a bulleted list with some definitions for these default tags?

Reply:
Ideally it would be great to put to a ref page like `pulumi stack tag` that would have this information; however, if that's not possible (because of auto-generated pages), then yeah, we should have them listed here. I'll track this as a follow-up issue.